pagopa/pn-frontend
# Fixed non-refreshing status enabled disabled app IO in contacts

## Short description
Fixed non-refreshing status enabled disabled app IO in contacts.

## List of changes proposed in this pull request
- Added useState vars instead of common const for status

## How to test
You should be able to see changing status enable / disable button in IO contac box now.
